src/memdb/memdb.ts

```diff
@@ -35,6 +107,50 @@ export const newMemDBPlugin = (): Plugin => {
       const filtered = filterTables(allTables, tables, skipTables, skipDependentTables);
       return await sync(memdbClient, filtered, stream, { deterministicCQId });
     },
+    write(stream: WriteStream): void {
+      stream.on('data', (request: pluginV3.cloudquery.plugin.v3.Write.Request) => {
```

Comment:
Can we wrap this like https://github.com/cloudquery/plugin-sdk-javascript/blob/f6413d255ae45c2a7dca451541d1ba7a85957254/src/grpc/plugin.ts#L7?

So we don't have to deal with the long name?